ember-primitives/rollup.config.mjs

```diff
@@ -15,7 +15,7 @@ export default {
   output: addon.output(),
   plugins: [
     addon.publicEntrypoints(['**/*.js']),
-    addon.appReexports(['components/*.js', 'helpers/**/*.js']),
+    addon.appReexports(['components/*.js', 'components/accordion/*.js', 'helpers/**/*.js']),
```

Comment:
maybe `components/accordion/index.js` instead of all the components in this folder? (renaming accordion/accordion to accordion/index?)

Reply:
How would you feel about me moving `components/accordion/accordion.gts` -> `components/accordion.gts` and keeping the "helper"/"sub"-components inside the `components/accordion` folder? That way people not using template-imports can use `<Accordion>` instead of `<Accordion::Accordion` and I can revert the change to the `appReexports`

Reply:
That would work as well! Good thinking!